### ISSUE TITLE
Dquartets Output error in Dsuite

### ISSUE THREAD

**Issue by ashnareyaz** [millanek/Dsuite#78]
Hello ,
I have been trying to use Dquartets and though everything worked fine and the program ran successfully but on completion the output files just had the heading-[P1      P2      P3      P4      Dstatistic      Z-score p-value f4-ratio        BBAA    ABBA    BABA] and no data . My SETS.txt file did not have any sample mentioned with Outgroup keyword , also it had 5 species with sample id and species id being tab separated. I'm not sure what went wrong, If you could please help me in this regard.

![Screenshot (209)](https://github.com/millanek/Dsuite/assets/90958868/ac5ce27a-7b8d-4561-9abf-6085810e840d)

**Comment by yangwukaidi** [comment]
hello,
I have the same problem as you. Do you have a solution for it?

**Comment by jqrichmond** [comment]
Same issue with me, exactly. This seems to be a bug. If anybody can give us a hand, I'd appreciate it. Thanks.

**Comment by wuyh0328** [comment]
I also encountered the same problem. Have you solved your problem?

**Comment by jqrichmond** [comment]
No, I had to give up and move on. I spent as much time as I could trying to figure it out.

**Comment by ShaolinXU** [comment]
Try commit: 4d7671c

```shell
git clone ...
cd Dsuite
git checkout 4d7671c
make 
```

Then run Dquartets as you want,

**Comment by jqrichmond** [comment]
Yes, thank you. I did this a while back and was able to run it no problem. Apologies – I thought I had commented on the github that I was able to get it to run using the update. -Jonathan

Jonathan Richmond, PhD
U.S. Geological Survey
4165 Spruance Rd. Suite 200
San Diego CA 92101
Phone: 619-225-6434
https://www.researchgate.net/profile/Jonathan-Richmond

From: Shaolin Xu ***@***.***>
Date: Saturday, July 20, 2024 at 1:50 PM
To: millanek/Dsuite ***@***.***>
Cc: Richmond, Jonathan Q ***@***.***>, Comment ***@***.***>
Subject: [EXTERNAL] Re: [millanek/Dsuite] Dquartets Output error in Dsuite (Issue #78)



 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



Try commit: 4d7671c<https://github.com/millanek/Dsuite/commit/4d7671c34709b84ec96fd1df27b2a3093388e8e3>

git clone ...

cd Dsuite

git checkout 4d7671c

make

Then run Dquartets as you want,
